### PR TITLE
amp-form: Explain that type=image is not allowed

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
I signed it!

File: extensions/amp-form/amp-form.md [hunk 1]
```diff
@@ -78,4 +78,4 @@ Action must be provided, `https` and is non-cdn link (does **NOT** link to https
 All other [form attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) are optional.
 
 ## Inputs
-Both `<input type=file>` and `<input type=password>` are not allowed. (This might be reconsidered in the future - please let us know if you require these and use cases).
+Currently, `<input type=file>`, `<input type=image>` and `<input type=password>` are not allowed. (This might be reconsidered in the future - please let us know if you require these and use cases).
```
Comment (mkhatib) [inline extensions/amp-form/amp-form.md]: Can you also mention `type=button` here as well.

**Comment by mkhatib** [comment]
LGTM one comment

**Comment by sebastianbenz** [comment]
Done. PTAL

**Comment by mkhatib** [comment]
Thanks!

**Pull request merged**